lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -209,8 +209,8 @@ class IndexQueryTestCase(ModuleStoreTestCase):
     NUM_PROBLEMS = 20
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 10, 147),
-        (ModuleStoreEnum.Type.split, 4, 147),
+        (ModuleStoreEnum.Type.mongo, 10, 149),
```

Comment:
This is another instance where the courseware tab's old functionality differs from the new course experience page. The sock runs a query to check the course verification state, the user's verification status and the upgrade deadline. There are other locations in the courseware page that run the same query, but it seems like as we transition to the course-experience code base it would be better to keep the functionality in there.